cmd/client/cmd/app.go

```diff
@@ -157,33 +157,53 @@ var appListCmd = &cobra.Command{
 	Short:   "List all apps",
 	Long:    "Return all apps with address and team.",
 	Example: "  $ teresa app list",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		tc := NewTeresa()
-		apps, err := tc.GetApps()
-		if err != nil {
-			if isNotFound(err) {
-				return newCmdError("You have no apps")
-			}
-			return err
-		}
-		// rendering app info in a table view
-		table := tablewriter.NewWriter(os.Stdout)
-		table.SetHeader([]string{"TEAM", "APP", "ADDRESS"})
-		table.SetRowLine(true)
-		table.SetAlignment(tablewriter.ALIGN_LEFT)
-		table.SetRowSeparator("-")
-		table.SetAutoWrapText(false)
-		for _, app := range apps {
-			a := ""
-			if len(app.AddressList) > 0 {
-				a = app.AddressList[0]
+	Run: appList,
+}
+
+func appList(cmd *cobra.Command, args []string) {
+	showApps, err := cmd.Flags().GetBool("show-apps")
```

Comment:
I have thought that `flags` was necessary to `app list` work. If I remove it how can I declare the `showApps` ?